packages/components/src/composite/current/stories/utils.tsx

```diff
@@ -0,0 +1,61 @@
+/**
+ * External dependencies
+ */
+import type { StoryContext } from '@storybook/react';
+
+/**
+ * Internal dependencies
+ */
+import type { CompositeStoreProps } from '..';
+
+export function UseCompositeStorePlaceholder( props: CompositeStoreProps ) {
+	return (
+		<dl>
+			{ Object.entries( props ).map( ( [ name, value ] ) => (
+				<>
+					<dt>{ name }</dt>
+					<dd>{ JSON.stringify( value ) }</dd>
+				</>
+			) ) }
+		</dl>
+	);
+}
+UseCompositeStorePlaceholder.displayName = 'useCompositeStore';
+
+export function transform( code: string, context: StoryContext ) {
+	// The output generated by Storybook for these components is
+	// messy, so we apply this transform to make it more useful
+	// for anyone reading the docs.
+	const config = ` ${ JSON.stringify( context.args, null, 2 ) } `;
+	const state = config.replace( ' {} ', '' );
+	return [
+		// Include a setup line, showing how to make use of
+		// `useCompositeStore` to convert store options into
+		// a composite store prop.
+		`const store = useCompositeStore(${ state });`,
+		'',
+		'return (',
+		'  ' +
+			code
+				// The generated output includes a full dump of everything
+				// in the store; the reader probably isn't interested in
+				// what that looks like, so instead we drop all of that
+				// in favor of the store generated above.
+				.replaceAll( /store=\{\{[\s\S]*?\}\}/g, 'store={ store }' )
+				// Now we tidy the output by removing any unnecesary
```

Comment:
Just spotted a typo as I was giving the PR another glance

```suggestion
				// Now we tidy the output by removing any unnecessary
```